application/classes/Ushahidi/Validator/Post/Create.php

```diff
@@ -227,6 +227,8 @@ public function checkTags(Validation $validation, $tags)
 
 	public function checkValues(Validation $validation, $attributes, $fullData)
 	{
+
+		$attributes = !empty($fullData['values']) ? $fullData['values'] : [];
```

Comment:
You can test it by creating a survey with a required task, create a post, mark the stage completed, save it, then from timeline or post detail try to publish it.